cfgov/unprocessed/js/owning-a-home/modules/explore-rates.js

```diff
@@ -0,0 +1,1224 @@
+    $( '.result-alert' ).not( '.credit-alert' ).remove();
+    $( '#dp-alert' ).remove();
+  }
+}
+
+
+function removeCreditScoreAlert() {
+  if ( $( '.credit-alert' ) || $( '.rangeslider__handle' ).hasClass( 'warning' ) ) {
+    $( '.rangeslider__handle' ).removeClass( 'warning' );
+    $( '.credit-alert' ).remove();
+  }
+}
+
+/**
+ * Add commas to numbers where appropriate.
+ * @param {string} value - Old value where commas will be added.
+ * @returns {string} value - Value with commas and no dollar sign.
+ */
+function addCommas( value ) {
+  value = unFormatUSD( value );
+  value = formatUSD( value, { decimalPlaces: 0 } )
+    .replace( '$', '' );
+  return value;
+}
+
+/**
+ * Initialize the range slider. http://andreruffert.github.io/rangeslider.js/
+ * @param {Function} cb - Optional callback.
+ * @returns {*} - Value of callback invocation.
+ */
+function renderSlider( cb ) {
+
+  $( '#credit-score' ).rangeslider( {
+    polyfill:    false,
+    rangeClass:  'rangeslider',
+    fillClass:   'rangeslider__fill',
+    handleClass: 'rangeslider__handle',
+    onInit:      function() {
+      slider.update();
+    },
+    onSlide:     function( position, value ) {
+      slider.update();
+    },
+    onSlideEnd:  function( position, value ) {
+      params.update();
+      if ( params['credit-score'] < 620 ) {
+        removeAlerts();
+        scoreWarning();
+      } else {
+        updateView();
+        removeCreditScoreAlert();
+      }
+    }
+  } );
+
+  if ( cb ) {
+    return cb();
+  }
+
+  return UNDEFINED;
+
+}
+
+/**
+ * Render chart data in an accessible format.
+ * @param {Object} data - Data processed from the API.
+ */
+function renderAccessibleData( data ) {
+  var $tableHead = $( '#accessible-data .table-head' );
+  var $tableBody = $( '#accessible-data .table-body' );
+
+  $tableHead.empty();
+  $tableBody.empty();
+
+  $.each( data.labels, function( index, value ) {
+    $tableHead.append( '<th>' + value + '</th>' );
+  } );
+
+  $.each( data.vals, function( index, value ) {
+    $tableBody.append( '<td>' + value + '</td>' );
+  } );
+}
+
+/**
+ * Render (or update) the Highcharts chart.
+ * @param  {Object} data Data processed from the API.
+ * @param  {Function} cb Optional callback.
+ * @returns {*} Value of callback invocation or undefined.
+ */
+function renderChart( data, cb ) {
+
+  if ( chart.isInitialized ) {
+
+    Highcharts.setOptions( {
+      xAxis: {
+        categories: data.labels
+      },
+      series: {
+        data: data.vals
+      }
+    } );
+
+    chart.$wrapper.removeClass( 'geolocating' );
+  } else {
+
+    if ( chart.$el.length < 1 ) {
+      return;
+    }
+
+    chart.$wrapper.addClass( 'geolocating' );
+
+    new Highcharts.Chart( {
+      chart: {
+        renderTo: chart.$el[0],
+        type: 'column',
+        animation: false
+      },
+      plotOptions: {
+        column: {
+          states: {
+            hover: {
+              color: '#2CB34A'
```

Comment:
Locally it doesn't look like either the green color specified in the highcharts theme file (`#ADDC91`)  or the hover color specified here (`#2CB34A`) are being applied to the columns. 

<img width="737" alt="screen shot 2017-08-10 at 7 47 55 am" src="https://user-images.githubusercontent.com/778171/29218472-c38d12ee-7e69-11e7-8f02-af8771bd061b.png">

The colors do get applied if I set the base column color in `plotOptions` here. I don't know if this is a local issue, or a difference between Highcharts versions...